exasol_advanced_analytics_framework/udf_communication/socket_factory/fault_injection.py

```diff
@@ -47,7 +50,7 @@ def _is_fault(self):
 
     def send(self, message: bytes):
         if self._is_fault():
-            self._logger.info("Fault injected", message=message)
+            self._logger.warning("Fault injected", message=message)
```

Comment:
Is this only for debugging or a permanent change?

Reply:
I want to make it a permanent change, because this is anyway for testing only and you should at least know which message were lost by the fault injector